resources/index.json

```diff
@@ -0,0 +1,4 @@
+{
+  "orgs": {},
+  "apps": []
+}
```

Comment:
The generated index file should not be committed to the repo, as we want CI to be able to update it on its own whenever there's a push to main. The index should only be bundled into the generated documentation site artifact and served by github pages.